qa/L0_client_memory_growth/client_memory_mail.py

```diff
@@ -38,7 +38,7 @@
     subject = "Triton Client Memory Growth " + sys.argv[1] + " Summary: " + today
     memory_graphs = glob.glob("client_memory_growth*.log")
     write_up = "<p>This test is run for both HTTP and GRPC protocols using C++ and Python test scripts. The max-allowed difference between mean and maximum memory usage is set to 10MB and 1MB for C++ and Python tests individually.</p>"
-    write_up += "<p><b>&#8226 What to look for</b><br>A linear memory growth in the beginning of the graph is acceptable only when it is followed by a flat memory usage. If a linear memory growth is observed during the entire test then there is possibly a memory leak.</p>"
+    write_up += "<p><b>&#8226 What to look for</b><br>The approximate time for each test case is 1.7 hours. A linear memory growth in the beginning of the graph is acceptable only when it is followed by a flat memory usage. If a linear memory growth is observed during the entire test then there is possibly a memory leak.</p>"
```

Comment:
Sorry, I wasn't specific enough before. Is it possible to update the script to actually measure the time for each test run, and then include that measured time in the information you output for each test just above the graph?

Reply:
Sure. Will update.